<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myConnect</class>
 <widget class="QDialog" name="myConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZBingMaYongTiS</family>
    </font>
   </property>
   <property name="title">
    <string>选择慈善家姓名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FZBingMaYongTiS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>卢本伟</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FZBingMaYongTiS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PDD</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FZBingMaYongTiS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大司马</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FZBingMaYongTiS</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎来到慈善模拟器—赌怪！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="add1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pp1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1111</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pp2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1115</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yess">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancell">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
